tests/autoyast/verify_ext4.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate autoyast_ext4 profile.
+#          * Verify registration
+#          * Verify partitioning: ext4 and swap
+#          * Verify software packages/patterns
+#          * Verify users
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use base 'basetest';
+use strict;
+use testapi;
+
+sub verify_registration {
+    my $output = script_output 'SUSEConnect --list-extensions';
+    unless ($output =~ /Basesystem.*Activated/ && $output =~ /Applications Server.*Activated/) {
+        die 'Registered system does not contains Basesystem and/or Applications Server';
+    }
+}
+
+sub verify_partitioning {
+    assert_script_run 'blkid /dev/vda | grep PTTYPE="dos"';
+    assert_script_run 'grep "swap\s\+swap" /etc/fstab';
+    assert_script_run 'grep "/\s\+ext4" /etc/fstab';
```

Comment:
Let's go a bit more strict here, checking that it's mount point for root partition. E.g. we could have used `findmnt` and validate output.